content/news/known-amendments.md

```diff
@@ -363,6 +394,20 @@ An address with a SignerList can disable the master key even if a regular key is
 * `tefBAD_AUTH_MASTER`
 
 
+## MultiSignReserve
+[MultiSignReserve]: #multisignreserve
+
+| Amendment ID                                                     | Status    |
+|:-----------------------------------------------------------------|:----------|
+| 586480873651E106F1D6339B0C4A8945BA705A777F3F4524626FF1FC07EFE41D | Planned   |
+
+Reduces the [owner reserve](reserves.html#owner-reserves) counted against your XRP Ledger account when it owns a [multi-signing](multi-signing.html) SignerList.
+
+Without this amendment, the owner reserve for a SignerList ranges from 15 to 50 XRP, depending on the number of signers in the list.
+
+With this amendment enabled, the owner reserve for a SignerList is 5 XRP, regardless of the number of signers. To receive the reduced reserve, you must wait for this amendment to be enabled and then use the [SignerListSet transaction](signerlistset.html) to create a SignerList, or replace or remove a SignerList that you created before the MultiSignReserve amendment was enabled.
```

Comment:
I'd probably lose the "or remove".  Perhaps something like, "... create a SignerList, or replace a SignerList that you created before ..."